src/util-bloomfilter.h

```diff
@@ -35,10 +35,7 @@ typedef struct BloomFilter_ {
 /* prototypes */
 BloomFilter *BloomFilterInit(uint32_t, uint8_t, uint32_t (*Hash)(const void *, uint16_t, uint8_t, uint32_t));
```

Comment:
the whole bloom filter code is unused I think, so might want to just remove all of it (separate commit)